### DISCUSSION TITLE
Is there already a way to build a "Notice extension"

### DISCUSSION THREAD

**Discussion by Pachat** [directus/directus#17373] [Feature Requests]
Motivation: In some cases, we need to display a notice that depends on a request.
No field in the database is required.
But the notice content will depend on several `/me` based database fields and collections in the component.

I could use a "fake" read-only textbox or wysiwyg, i.e. a never-updated field.
But I think a true "extension of notice" (that can interpret HTML) would be more appropriate.

**Comment by rijkvanzanten** [comment]
Heya! Thank you for taking the time to submit this request! 

It has been over **90 days**, and this discussion has not received at least **15 votes** from the community. This means that we don't feel like there's enough community interest to warrant further R&D into this topic at this time. :ice_cube:

This request will now be closed to keep our discussions tidy. Please reach out if you have any questions!

_For more information, see our [Feature Request Process](https://docs.directus.io/contributing/feature-request-process.html)._
